src/foam/nanos/google_api/google_api_auth/GoogleApiCredentials.js

```diff
@@ -5,7 +5,7 @@
  */
 
 foam.CLASS({
-  package: 'foam.nanos.export',
+  package: 'foam.nanos.google_api',
```

Comment:
I would prefer 
foam.nanos.google.api 
and google.api.auth if you need it. 
google.api.sheets...

Reply:
fixed